Libraries/Hotcakes.Commerce/Orders/LineItem.cs

```diff
@@ -590,6 +666,17 @@ public string DiscountDetailsAsHtml()
             return sb.ToString();
         }
 
+        /// <summary>
+        ///     Generates a line by line summary of the upcharge that are in this line item.
+        /// </summary>
+        /// <returns>String - an HTML version of the line items for display to the merchant.</returns>
+        public string UpchargeDetailsAsHtml()
+        {
+            var sb = new StringBuilder();
+            sb.Append($"Upcharge Amount: ({TotalUpcharge():C})<br />");
```

Comment:
@ArielBlanco1990 This needs to be localized.

Reply:
@WillStrohl This is now resolved.